templates/html_report.template

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
```

Comment:
Maybe we should make it clear that this is template for interop tests by renaming this file to "interop_html_report.template" or sth like that?